### DISCUSSION TITLE
The "send feedback" links are somewhat broken

### DISCUSSION THREAD

**Discussion by sgbaird** [community/community#17895] [Other Feature Feedback, Questions, & Ideas]
The "send feedback" links are somewhat broken. https://github.com/github/feedback/discussions/categories/feed-feedback

It should be https://github.com/github/feedback/discussions/categories/feed

![image](https://user-images.githubusercontent.com/45469701/171969744-bb987ef0-1072-4317-a821-c292c575f875.png)

**Comment by sgbaird** [comment]
Nature of question updated. This made it difficult to realize that my issue was already mentioned in several discussion posts in the feed category. Especially since if you search without changing the category back to `category:feed` as it should be, it comes up with "no matching discussions".

![image](https://user-images.githubusercontent.com/45469701/171969901-85c94254-a414-4998-8d7f-be040362f3e4.png)

**Comment by siminapasat** [comment]
Thanks for raising this @sgbaird 🙏 We'll have a look!

**Reply by siminapasat** [reply]
(Fixed now 😊 )
